packages/web-comment/src/routes/comment-appeal/appeals/index.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const { getOpenAppeals, getClosedAppeals } = require('../../../utils/appealSorting');
+const { formatAddress } = require('../../../utils/formatAddress');
+const { getPostcodeSearchResults } = require('../../../utils/getPostcodeSearchResults');
+
+const router = express.Router();
+
+router.get('/', async (req, res) => {
+	const postcode = req.query.search;
+	const postcodeSearchResults = await getPostcodeSearchResults({
+		postcode,
+		'with-appellant': true
+	});
+
+	if (!postcodeSearchResults.length) {
+		return res.redirect(`/comment-appeal/appeal-search-no-results?search=${postcode}`);
+	}
+
+	postcodeSearchResults.forEach((appeal) => (appeal.formattedAddress = formatAddress(appeal)));
+
+	const openAppeals = getOpenAppeals(postcodeSearchResults);
+	openAppeals.sortByInterestedPartyRepsDueDate(openAppeals);
+
+	const closedAppeals = getClosedAppeals(postcodeSearchResults);
+	closedAppeals.sortByCaseReference();
+
+	res.render(`comment-appeal/appeals/index`, { postcode, openAppeals, closedAppeals });
```

Comment:
Be good to use a pattern we have elsewhere for this. Have this in a `controller.js` file, then import into the router and wrap with `asyncHandler` - that way if any promises reject, the error gets handled.

e.g. see https://github.com/Planning-Inspectorate/appeal-planning-decision/commit/2b22d191344252d276f454b341266464306d71dc

Reply:
I'd intended just a controller file in each route folder, I will refactor!